controller/search.go

```diff
@@ -243,3 +247,34 @@ func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
 
 	return ctx.OK(&response)
 }
+
+// Iterate over the WI list and read parent IDs
+// Fetch and load Parent WI in the included list
+func (c *SearchController) enrichWorkItemList(ctx *app.ShowSearchContext, res *app.SearchWorkItemList) {
+	fetchInBatch := []uuid.UUID{}
+	for _, wi := range res.Data {
+		if wi.Relationships != nil && wi.Relationships.Parent != nil && wi.Relationships.Parent.Data != nil {
+			parentID := wi.Relationships.Parent.Data.ID
+			fetchInBatch = append(fetchInBatch, parentID)
+		}
+	}
+	wis := []*workitem.WorkItem{}
+	err := application.Transactional(c.db, func(appl application.Application) error {
+		var err error
+		wis, err = appl.WorkItems().LoadBatchByID(ctx, fetchInBatch)
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
I think this can be shrinked to just `return err`. If `err` is `nil`, we might as well return it ;)

Reply:
yes, good catch!  done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/f267820f7c360f8da54ca40e6e34d6399f38f77e